libs/gi/ui/src/util/getDisplayUtil.tsx

```diff
@@ -114,6 +115,7 @@ export function getDisplaySections(data: UIData): DisplaySections {
   )
   return [
     ...basic,
+    ['character', data.getCharacterStats()],
```

Comment:
Character header still shows even if nothing is there (like when compare is unselected)
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/9e537b16-5928-4dd1-a412-106f4e4a5795)

Reply:
Yeah I had an attempt to fix it: https://github.com/frzyc/genshin-optimizer/blob/c48afc3c4e2db91e074a30cc53bca773511dad73/libs/gi/ui/src/components/character/StatDisplayComponent.tsx#L116

But its actually a lot more complicated to execute